trace_helper.cpp

```diff
@@ -15,80 +15,83 @@
  * limitations under the License.
  */
 
-#include "Serial.h"
-#include "rtos/Mutex.h"
+#include "drivers/Serial.h"
+
+/**
+ * Serial object for console tracing
+ */
+mbed::Serial pc(USBTX, USBRX);
```

Comment:
Remove pc.baud(115200) from main() and change this as:
mbed::Serial pc(USBTX, USBRX, 115200);